gen-imprecise.js

```diff
@@ -0,0 +1,24 @@
+const plural = require('plural');
+
+const imprecise_units = [
+  "bag", "bar", "bottle", "bowl", "breast", "bulb", "bun", "bunch", "can",
+  "carton", "cone", "clove", "large", "medium", "mini", "small",
+  "cube", "fillet", "head", "jar", "package", "packet", "pack", "patty",
+  "piece", "portion", "servings", "roll", "slice", "handful", "pinch",
+  "touch", "envelope", "sprig", "sheet"
+];
+
+const l = imprecise_units.concat(['dash','splash','unit_', "container_"]).join(' / ');
```

Comment:
these are the units that have a buggy plural or that have a conflict with another rule in the grammar. So I consider those as special cases. The concatenation is because they end up all in the same line and this is how you separate entries in pegjs